sxs/waveforms/alignment.py

```diff
@@ -1,12 +1,13 @@
 import numpy as np
 
+import quaternion
```

Comment:
Everything else in this package is using the package quaternionic instead of quaternion...